application/testing/CMakeLists.txt

```diff
@@ -992,6 +1002,22 @@ if(F3D_PLUGIN_BUILD_ALEMBIC AND F3D_PLUGIN_BUILD_ASSIMP)
   f3d_test(NAME TestReadersListMultiplePlugins ARGS --readers-list --load-plugins=assimp,alembic NO_BASELINE REGEXP_FAIL "Plugin failed to load")
 endif()
 
+# Test rendering backends
+f3d_test(NAME TestRenderingBackendAuto DATA cow.vtp RENDERING_BACKEND auto)
+if(UNIX AND NOT APPLE AND VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240914)
+  f3d_test(NAME TestRenderingBackendEGL DATA cow.vtp RENDERING_BACKEND egl ARGS --verbose REGEXP "vtkEGLRenderWindow")
```

Comment:
REGEXP tests should not have baselines afair. In that case, I would remove the REGEXP

Reply:
Actually, I test two things here:
- Make sure the backend is correct by checking the regex
- Validate that the rendering is correct with that backend

So I'd say that a regexp with a baseline could be ok.

Reply:
that would be nice except that the regexp results override the baseline result afair

Reply:
so I need to duplicate the test?  
let me take a look to see if I can make it work.

Reply:
duplicating is a way to do it yes